### PR TITLE
7.3.0 Release

### PR DESCRIPTION
**Pull request by UTDZac**
Patch Notes:
- [Custom Code Extensions](https://user-images.githubusercontent.com/4258818/218183892-744fab56-6ed0-4797-bf7e-3c313cc017ef.png) are now supported!
   - Anyone can create and share custom extensions that integrate with the Tracker code
   - Try out some user-created extensions by visiting our [Extensions wiki page](https://github.com/besteon/Ironmon-Tracker/wiki/Tracker-Add-ons#custom-code-extensions).
- [Type Defenses](https://user-images.githubusercontent.com/4258818/211697610-55d5462c-a5ad-4231-9e0f-e495d18d2b75.png) for each Pokémon can now by seen by clicking the "Types" under the Pokémon's icon
- Officially added a [LICENSE](https://github.com/besteon/Ironmon-Tracker/blob/main/LICENSE.txt) file to the project
- Can now manually create restore points using the Time Machine
   - Note: restore points only exist for the current play session
- Fixed some Log Viewer bugs for Emerald Rival 2 inaccuracies and Mr. Mime not appearing on trainers
- Fixed a bug where Future Sight, Doom Desire, and Beat Up incorrectly calculated move type effectiveness (now always 1.0x)
- Fixed a bug where Super Potion was incorrectly calculated as a 60 HP healing item
- Fixed a bug where certain item names wouldn't render correctly due to non-English characters
- [Bizhawk 2.9] Fixed a bug where the ColorPicker window would stop working when editing Theme colors
- [MGBA] Fixed a bug where Quickloading using premade seeds would sometimes fail

### CODE REVIEW

File: extensions/How to use.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+------------------------
+Extensions allow anyone to write custom code that integrates with the Ironmon Tracker script.
+These extensions can be anything, from new features or improvements to existing features.
+Developers can create and share their own extensions with ease.
+
+To learn more, visit the wiki: https://github.com/besteon/Ironmon-Tracker/wiki/Tracker-Add-ons#custom-code-extensions
+
+To add a custom extension to your Tracker:
+- Simply place the extension's files inside the "extensions" folder found in your Tracker folder.
+- Enable or disable extensions from within the Tracker by navigating to Settings -> Setup -> Extensions.
```
Comment (Fellshadow) [inline extensions/How to use.txt:12]: This line needs updated to reflect the menu change

**Review by Fellshadow — CHANGES_REQUESTED** [review]
One quick thing, then it's good to go

File: extensions/How to use.txt (hunk 1 above)
Comment (UTDZac) [reply extensions/How to use.txt]: Good catch. Fixed.

**Review by Fellshadow — APPROVED** [review]
All good, whenever you're ready to go 👍

**Pull request merged**